src/wechaty/plugin.py

```diff
@@ -0,0 +1,325 @@
+"""
+Python Wechaty - https://github.com/wechaty/python-wechaty
+
+Authors:    Huan LI (李卓桓) <https://github.com/huan>
+            Jingjing WU (吴京京) <https://github.com/wj-Mcat>
+
+2020-now @ Copyright Wechaty
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from __future__ import annotations
+
+from abc import abstractmethod, ABCMeta
```

Comment:
If I remember right, the Python 3.7 or above does not need the `ABC` anymore?

Reply:
Ok, this is my negligence, I will move it later.

Reply:
There are some ways to ceate `abstract class`.

```python
from abc import ABC

class MyABC(ABC):
    pass
```

and 
```python
from abc import ABCMeta

class MyABC(metaclass=ABCMeta):
    pass
```

 If we don't use `ABC`, how to create abstract class?  Do I understand this correctly ?